lib/task.js

```diff
@@ -109,11 +114,47 @@ function factory(
         self.renderContext.api.files = self.renderContext.api.base + '/files';
         self.renderContext.api.nodes = self.renderContext.api.base + '/nodes';
 
-        self.renderOwnOptions(_.cloneDeep(self.definition.options));
-
         return self;
     }
 
+    Task.prototype.getSkuId = function() {
+        var skuId;
+        if(!this.nodeId)  {
+            return Promise.resolve();
+        }
+
+        return waterline.nodes.needByIdentifier(this.nodeId)
+            .then(function(node) {
+                if (node.sku) {
+                    skuId = node.sku;
+                }
+                return skuId;
+            });
+    };
+
+    Task.prototype.renderAll = function(){
+        var self = this;
+        return self.getSkuId(this.nodeId).
+            then(function (skuid) {
+                if (!skuid){
+                    return env.get('config', {}, ['global']).
```

Comment:
Using the constant is better than hard-coded the string here: https://www.github.com/RackHD/on-core/blob/master/lib/common/constants.js#L281